packages/nextra-theme-docs/tailwind.config.ts

```diff
@@ -52,17 +52,17 @@ export default {
       blue: colors.blue,
       yellow: colors.yellow,
       primary: {
-        50: makePrimaryColor(97),
-        100: makePrimaryColor(94),
-        200: makePrimaryColor(86),
-        300: makePrimaryColor(77),
-        400: makePrimaryColor(66),
-        500: makePrimaryColor(50),
-        600: makePrimaryColor(45),
-        700: makePrimaryColor(39),
-        750: makePrimaryColor(35),
-        800: makePrimaryColor(32),
-        900: makePrimaryColor(24)
+        50: makePrimaryColor(52),
+        100: makePrimaryColor(49),
+        200: makePrimaryColor(41),
+        300: makePrimaryColor(32),
+        400: makePrimaryColor(21),
+        500: makePrimaryColor(5),
+        600: makePrimaryColor(0),
+        700: makePrimaryColor(-6),
+        750: makePrimaryColor(-10),
+        800: makePrimaryColor(-13),
+        900: makePrimaryColor(-11)
```

Comment:
hi @wayne-shih, I think the last number, a little typo, should be `-21`?

<img width="393" alt="image" src="https://github.com/user-attachments/assets/5bcad9e8-8081-4576-a319-6287b9a9ec4c">

Reply:
whoops! yeah typo 
good catch, want me to do a patch?

Reply:
fixed here : https://github.com/shuding/nextra/pull/3656
thanks!

Reply:
thank you for super fast fix :)